pyasic/config/fans.py

```diff
@@ -50,6 +50,17 @@ def as_am_modern(self) -> dict:
     def as_bosminer(self) -> dict:
         return {"temp_control": {"mode": "auto"}}
 
+    def as_epic(self) -> dict:
+        return {
+            "fans": {
+                "Auto": {
```

Comment:
For Auto we need to set a Target Temperature. Will check what happens if you send a null value but generally the expectation is something like shutdown-15 degrees
   "Auto": {
      "Idle Speed": 100,
      "Target Temperature": 60
    }

Reply:
Added a default for this, it is parsed in `TemperatureConfig`, then merged in with this config with `merge_dicts`.